scripts/linters/general_purpose_linter.py

```diff
@@ -129,18 +128,6 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
-    {
-        'regexp': re.compile(r'bypass'),
-        'message': 'The use of the word "bypass" is not allowed, ' +
-                   'particularly with regards to bypassSecurityTrustHTML() ' +
-                   'and similar functions in Angular.',
-        'excluded_files': (
-            warranted_angular_security_bypasses
-            .EXCLUDED_BYPASS_SECURITY_TRUST_FILES),
```

Comment:
Remove these files from the codebase, if it's not needed anymore?

Reply:
Done